Library/Homebrew/rubocops/cask/array_alphabetization.rb

```diff
@@ -0,0 +1,38 @@
+# typed: true
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Cask
+      class ArrayAlphabetization < Base
+        extend AutoCorrector
+
+        def on_send(node)
+          return if node.method_name != :zap
+
+          node.each_descendant(:pair).each do |pair|
+            pair.each_descendant(:array).each do |array|
+              if array.children.length == 1
+                add_offense(array, message: "Remove the `[]` around a single `zap trash` path") do |corrector|
```

Comment:
I believe this will capture all zap methods, not just `trash`, in which case the message should be update with the method name.

Reply:
TIL there are other `zap` methods, the ones I eyeballed just had `trash`. I will look into this!

Reply:
Ah, yes, I see them now in the failing CI syntax job. 😭 I will restrict this to `trash` only then since the others it looks like they're commands or something where the ordering matters and can't always be alphabetical? And the original issue was for `zap trash: []`.